ref_ts/static/ref_ts/d3/LICENSE

```diff
@@ -1,26 +0,0 @@
-Copyright (c) 2010-2014, Michael Bostock
```

Comment:
We should include D3 in hs_core's static files as multiple resources are likely to use it.  Otherwise, we risk having multiple versions from different resources, which may lead to conflicts that are hard to debug.